ml-agents/README.md

```diff
@@ -135,3 +144,30 @@ variable named `env` in this example, can be used in the following way:
 - **Close : `env.close()`**
   Sends a shutdown signal to the environment and closes the communication
   socket.
+
+
+
+## `mlagents.trainers`
+
+1. Open a command or terminal window.
+2. Run 
+
+```
+mlagents-learn <trainer-config-path> --run-id=<run-identifier> --train <environment-name>
+```
+
+   Where:
+    - `<trainer-config-path>` is the relative or absolute filepath of the
+      trainer configuration. The defaults used by environments in the ML-Agents
+      SDK can be found in `config/trainer_config.yaml`.
+    - `<run-identifier>` is a string used to separate the results of different
+      training runs
+    - The `--train` flag tells `mlagents-learn` to run a training session (rather
+      than inference)
+    - `<environment-name>` __(Optional)__ is the path to the Unity executable you want		to train. __Note:__ If this argument is not passed, the training will be 
```

Comment:
It looks like there's a hard tab in the middle of this line. Was that intentional?

Reply:
It was not